docker/centos-7/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM centos/devtoolset-4-toolchain-centos7:latest
+LABEL maintainer Apache Geode <dev@geode.apache.org>
+
+USER root
+WORKDIR /
+
+RUN yum update -y && \
+    yum -y clean all
+
+COPY bellsoft.repo /etc/yum.repos.d/
+
+RUN yum update -y && \
+    yum install -y \
+        git \
+        make \
+        zlib-devel \
+        patch \
+        openssl-devel \
+        bellsoft-java11 \
+        doxygen \
+        python3-pip \
+        which && \
+    yum -y clean all
+
+RUN pip3 install --upgrade pip && \
+    pip3 install cpp-coveralls
+
+RUN installer=$(mktemp) \
+    && curl -o ${installer} -L $(curl -s https://api.github.com/repos/Kitware/CMake/releases/latest \
+        | grep 'browser_download_url.*Linux-x86_64\.sh' \
+        | cut -d : -f 2,3 \
+        | tr -d '"') \
+    && bash ${installer} --skip-license --prefix=/usr/local \
+    && rm ${installer}
+
+ARG GEODE_VERSION=1.12.0
```

Comment:
Why is this not using the latest geode of 1.13.1?  This seems to be the case in all of the dockerfiles added.  How do we plan to keep this up to date?  It seems painful to remember to update this value in all dockerfiles every time there is a release, but maybe that is more of a wait and see problem

Reply:
Good catch. Yah, we have several places where we set GEODE_VERSION. Seems like this should be set in a single place. For now, setting  all to 1.13.1. Will create a ticket to fix this.

Reply:
For the dockerfiles, I'm pretty sure on the `docker build` command you can pass args.  So if scripting all those `docker build` commands, you would be able to have a single variable reused

Reply:
Captured the GEODE_VERSION issue as GEODE-8954. Also, put in your idea of passing in the variable to the docker command.